.github/workflows/nodejs.yaml

```diff
@@ -0,0 +1,22 @@
+name: Node.js CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x]
```

Comment:
shall we simply set one node version? perhaps you also want to add this version to the package.json `engines`.

Reply:
Which version should we use? 15?

Reply:
Yep, general rule newest stable one, 12 is good.